site-src/index.md

```diff
@@ -20,54 +20,119 @@ variety of networking use-cases.
 
 ![Gateway API Model](./images/api-model.png)
 
-
-How do Gateway API improve upon current standards like Ingress?
-
-- **More expressive** - They express more core functionality for things like
-header-based matching, traffic weighting, and other capabilities that were only
-possible in Ingress through custom means.
-- **More extensible** - They allow for custom resources to be linked at various
-layers of the API. This allows for more granular customization at the
-appropriate places within the API structure.
-- **Role oriented** - They are separated into different API resources that map
-to the common roles for running applications on Kubernetes.
-- **Generic** - This isn't an improvement but rather something
+Following are some of the design goals where Gateway aims to improve upon 
+current standards like Ingress.
+
+- **Role-oriented** - Gateway is composed of API resources which model 
+organizational roles that use and configure Kubernetes service networking. 
+A role-oriented API allows diverse teams to operate on shared network 
+infrastructure. Gateway is designed to balance the flexibility and control of 
+these roles in a way that reflects the culture and structure of the organization
+ using it.
```

Comment:
Although this content is good, my personal preference would be to keep content like this a bit more brief on the main page.